src/nodes/typeGuards/union.ts

```diff
@@ -300,7 +304,15 @@ export function isHasJSDoc(node: ts.Node): node is ts.HasJSDoc {
 		ts.isExportSpecifier(node) ||
 		ts.isCaseClause(node) ||
 		isEndOfFileToken(node)
-	);
+	) {
+		return true;
+	}
+
+	if (semver.satisfies(ts.version, ">=4.4")) {
+		return ts.isClassStaticBlockDeclaration(node);
```

Comment:
Alternatively to this, we could do `isClassStaticBlockDeclaration in ts`

Pros:
- No dependencies required

Cons
- May end up calling private API functions

Reply:
We also have the option of creating our own `isTsVersionAtLeast(magor: number, minor: number)` function.